Source/Krypton Components/Krypton.Toolkit/Controls Toolkit/KryptonDataGridViewCheckBoxCell.cs

```diff
@@ -265,7 +265,7 @@ private ButtonState ButtonStateInternal
                 }
 
                 // Grab the internal property implemented by base class
-                return (ButtonState)_piButtonState.GetValue(this, null);
+                return (ButtonState)_piButtonState?.GetValue(this, null)!;
```

Comment:
Should these be using `as` like the other changes above ?

Reply:
Used `is` & other ReSharper suggestions